contracts/0.4.24/Lido.sol

```diff
@@ -210,6 +210,7 @@ contract Lido is Versioned, StETHPermit, AragonApp {
     {
         _bootstrapInitialHolder();
         _initialize_v2(_lidoLocator, _eip712StETH);
```

Comment:
Can we remove _initialize_v2 altogether by moving it's logic to v3?

Reply:
We can, but should we?